src/pybind/rados/rados.pyx

```diff
@@ -2046,6 +2048,36 @@ cdef class WriteOp(object):
         with nogil:
             rados_write_op_set_flags(self.write_op, _flags)
 
+    @requires(('xattr_name', str_type), ('xattr_value', bytes))
+    def setxattr(self, xattr_name, xattr_value):
+        """
+        Set an xattr
+        :param xattr_name : name of the xattr
```

Comment:
can you remove the extra whitespace from parameters, something like `:param xattr_name:`

Reply:
@ideepika updated. please check.

Reply:
thanks a lot